polar-core/src/bindings.rs

```diff
@@ -425,52 +427,91 @@ impl BindingManager {
     }
 }
 
+fn combine(left: &Symbol, l: Operation, right: &Symbol, r: Operation) -> Operation {
+    let unis = l
```

Comment:
```suggestion
    let unifications = l
```

would make it easier for me to read, I'd know what it is immediately rather than thinking for a few